source/jormungandr/jormungandr/parking_space_availability/car/parking_places.py

```diff
@@ -41,7 +41,7 @@ def __init__(self, available=None, occupied=None, available_PRM=None, occupied_P
         if occupied_PRM is not None:
             self.occupied_PRM = occupied_PRM
 
-        if any(True for n in [available, occupied, available_PRM, occupied_PRM] if n is not None):
+        if any(n is not None for n in [available, occupied, available_PRM, occupied_PRM]):
             self.total_places = (available or 0) + (occupied or 0) + (available_PRM or 0) + (occupied_PRM or 0)
```

Comment:
Not sure it's the scope of this PR: to me, None is "no information" while 0 is no place.
But it was there before, so unless someone else thinks this can be a problem (those `or 0`), let's keep it that way.

Reply:
As discussed, this does what is expected. 0 (no place) will be returned.